.eslintrc.js

```diff
@@ -33,9 +33,23 @@ const restrictedImportColors = [
 ];
 
 module.exports = {
-  plugins: ['prettier', 'import', 'rulesdir', '@typescript-eslint'],
+  root: true,
```

Comment:
The changes in this file is mostly a copy-paste from [eslint-config-react-app](https://github.com/facebook/create-react-app/tree/main/packages/eslint-config-react-app). Instead of `extending` the config - we now fully maintain it on our end.